cypress/integration/settings/email-change.js

```diff
@@ -0,0 +1,45 @@
+/* global cy */
+describe('Email input field', () => {
+  before(() => {
+    cy.login();
+    cy.visit('/settings');
+  });
+
+  it('Should be possible to type new email', () => {
+    cy.get('[id=new-email]')
+      .type('bar@foo.com')
+      .should('have.attr', 'value', 'bar@foo.com');
+  });
+
+  it('Should be possible to confirm email', () => {
+    cy.get('[id=confirm-email]')
+      .type('bar@foo.com')
+      .should('have.attr', 'value', 'bar@foo.com');
+  });
+
+  it('Should be possible to save new email', () => {
+    cy.get('[id=form-update-email]').within(() => {
+      cy.contains('Save').click();
+    });
+  });
```

Comment:
I think these can be combined into a single test.  The rationale is that they're a bit repetitive and single large tests are more efficient (fewer setup/teardown steps) than multiple small ones.  Since Cypress tells you exactly where you failed, it's still pretty informative.